### PR TITLE
config.tf: updated version string for 1.7.3 patch release

### PR DESCRIPTION
**Pull request by Quentin-M**

### CODE REVIEW

File: config.tf [hunk 1]
```diff
@@ -75,7 +75,7 @@ variable "tectonic_versions" {
     kubernetes    = "1.7.3+tectonic.1"
```
Comment (diegs) [inline config.tf]: bump to `1.7.3+tectonic.3`

Comment (Quentin-M) [reply config.tf]: Do we need to do that usually? I think this is to indicate hyperkube bumps if I remember clearly.

Comment (diegs) [reply config.tf]: this tells the KVO which version payload to use

Comment (Quentin-M) [reply config.tf]: Ack. Thanks!

Comment (diegs) [reply config.tf]: also the KVO version will be `quay.io/coreos/kube-version-operator:v1.7.3-kvo.4`

**Comment by Quentin-M** [comment]
.4 is the KVO image but all the rest of the strings are .3 :)

**Comment by Quentin-M** [comment]
test upgrade from 1.7.3-tectonic.2 1.7.3-tectonic.2

**Pull request merged**